c7n/filters/securityhub.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from c7n.utils import local_session, chunks, type_schema
+from .core import Filter
+from c7n.manager import resources
+
+
+class SecurityHubFindingFilter(Filter):
+    """Check if there are Security Hub Findings related to the resources
+    """
+    schema = type_schema(
+        'finding',
+        types={'type': 'array', 'items': {'type': 'string'}}
+    )
+    permissions = ('securityhub:GetFindings',)
+
+    def process(self, resources, event=None):
+        client = local_session(self.manager.session_factory).client(
+            'securityhub', region_name='us-east-1')
+
+        if self.manager.data['resource'] in {'app-elb'}:
```

Comment:
this filter feels like it started with copy paste from the health filter, which had a specific reasoning on the special treatment of alb, which isn't appropriate here.

Reply:
guilty as charged.  I refactor that out.